### ISSUE TITLE
[FEAT] Change GitHub Pages theme to simple theme

### ISSUE THREAD

**Issue by vikasganiga05** [collab-community/coding-youtubers#30]
My suggestion is to change the GitHub pages theme to a simple & clean theme. The current theme is good but not good typography. So, some people will find it hard to read.

**Comment by KendallDoesCoding** [comment]
> My suggestion is to change the GitHub pages theme to a simple & clean theme. The current theme is good but not good typography. So, some people will find it hard to read.

True, can you give me a name of the theme or make a PR of what looks good to you? and then, we will take it from there.

**Comment by vikasganiga05** [comment]
> > My suggestion is to change the GitHub pages theme to a simple & clean theme. The current theme is good but not good typography. So, some people will find it hard to read.
> 
> True, can you give me a name of the theme or make a PR of what looks good to you? and then, we will take it from there.

Okay, I will soon update you.

**Comment by vikasganiga05** [comment]
What are your thoughts on this theme &rarr; https://pages-themes.github.io/cayman/
And it's docs &rarr; https://github.com/pages-themes/cayman

**Comment by KendallDoesCoding** [comment]
Looks better, but we require approval from the team. I approve, We require
approval from Agrim and Adarsh.

**Comment by vikasganiga05** [comment]
Yes, let's wait for their approval or any other suggestion from &rarr; https://github.com/pages-themes

**Comment by Adarsh-jaiss** [comment]
-- i have checked out themes of that repository, but i love the current theme more :) 

I think we don't need any changes

**Comment by KendallDoesCoding** [comment]
> -- i have checked out themes of that repository, but i love the current theme more :)
> 
> -- I think we don't need any changes. The current theme is awesome :)

cc: @vikasganiga05

**Comment by Agrim-Sharma174** [comment]
> -- i have checked out themes of that repository, but i love the current theme more :)
> 
> -- I think we don't need any changes. The current theme is awesome :)

IMO too the current one looks better

**Comment by KendallDoesCoding** [comment]
2 people agree, 2 people disagree, but the founder disagrees so don't think we'll do this for now 

cc: @vikasganiga05

**Comment by vikasganiga05** [comment]
Okay, Let's go with the current theme :slightly_smiling_face:

**Comment by KendallDoesCoding** [comment]
Hi, I think we really need to change the theme because the description is taking up many lines, @vikasganiga05 can you deploy this repo with cayman in your fork and show me how it'll look?

**Comment by AnimeshKumar923** [comment]
> Hi, I think we really need to change the theme because the description is taking up many lines, @vikasganiga05 can you deploy this repo with cayman in your fork and show me how it'll look?

I support this.

If possible use this font for the banner of website:

https://fonts.google.com/share?selection.family=Roboto:wght@500

**Comment by vikasganiga05** [comment]
> Hi, I think we really need to change the theme because the description is taking up many lines, @vikasganiga05 can you deploy this repo with cayman in your fork and show me how it'll look?

Okay, Done :+1: Checkout here &rarr; https://vikasganiga05.github.io/coding-youtubers/

**Comment by KendallDoesCoding** [comment]
Yeah. It really looks much better ngl

**Comment by AnimeshKumar923** [comment]
@vikasganiga05 hey, can you make the table borders a little darker?

It would look much better.

**Comment by KendallDoesCoding** [comment]
> @vikasganiga05 hey, can you make the table borders a little darker?
> 
> It would look much better.

@AnimeshKumar923 Please make a issue on that.

**Comment by vikasganiga05** [comment]
> 

Yeah!

**Comment by fluentmoheshwar** [comment]
I think this is perfect for this repo https://github.com/dracula/gh-pages
@KendallDoesCoding @Adarsh-jaiss @vikasganiga05
